airbyte-integrations/connectors/source-s3/pyproject.toml

```
@@ -22,10 +22,15 @@ wcmatch = "==8.4"
 dill = "==0.3.4"
 transformers = "4.38.2"
 urllib3 = "<2"
-airbyte-cdk = {extras = ["file-based"], version = "^5.7.4"}
+numpy = "<2.0.0"  # TODO: Remove after CDK is published
 
-[tool.poetry.scripts]
-source-s3 = "source_s3.run:run"
+[tool.poetry.dependencies.airbyte-cdk]
+extras = ["file-based",]
+version = "^5.7.4"
```

Might as well update the explicit piece here
```suggestion
version = "^5.14.0"
```

---

YES! There was a critical CDK update in 5.7 that drove me to make sure this was the minimum - because otherwise a version conflict was forcing it silently to an older CDK version. Happy to bump to the latest, and will apply this suggestion. 👍